### PR TITLE
fix(com): fix iframe initializer handling of src parameter

### PR DESCRIPTION
**Pull request by tomermes**
The previous version never used the src that was passed to it, only as a condition. See the compiled version:
image

### CODE REVIEW

**Comment by AviVahl** [comment]
Test? What does it fix?

**Comment by AviVahl** [comment]
Btw using || instead of ?? feels wrong, that's why I'm asking.

File: packages/core/src/com/initializers/iframe.ts [hunk 1]
```diff
@@ -40,19 +40,19 @@ export function deferredIframeInitializer({ communication: com, env: { env, endp
                 envReadyPromise,
                 instanceId,
                 src:
-                    src ?? managed
+                    src || managed
```
Comment (RomanYarik) [inline packages/core/src/com/initializers/iframe.ts:43]: maybe 
```suggestion
                    src ?? (managed ? ... )
```

**Review by RomanYarik — APPROVED** [review]

**Comment by RomanYarik** [comment]
please add a description describing why this fix was needed

**Comment by tomermes** [comment]
> Btw using || instead of ?? feels wrong, that's why I'm asking.

Committed Roman's suggestion to add "(" instead, and added a description to what it fixes.

**Comment by AviVahl** [comment]
What about a test? This can regress in the future, if not covered.

**Comment by RomanYarik** [comment]
@AviVahl it wasn't covered previously either, since we don't have tests which checks the src...

opening a PR for testing this

**Pull request merged**